src/status_im/commands/handlers/jail.cljs

```diff
@@ -79,31 +79,31 @@
 
 ;; TODO(alwx): rewrite
 (reg-handler :command-handler!
-  (after (print-error-message! "Error on command handling"))
-  (handlers/side-effect! command-handler!))
+             (after (print-error-message! "Error on command handling"))
+             (handlers/side-effect! command-handler!))
```

Comment:
seems like too much indent

Reply:
that's actually correct indent https://github.com/bbatsov/clojure-style-guide#vertically-align-fn-args

Reply:
@rasom correct, i was confused comparing with other `reg-handler` calls where args placed on second line